### ISSUE TITLE
No username or password creation upon wallet setup (android)

### ISSUE THREAD

**Issue by sheadawson** [polkawallet-io/polkawallet-flutter#203]
Hi, I have had an experience similar to #159, but on Android. Please read this full report before assuming I just lost my password :)

After setting up the app on 3 September and transferring DOT to the address, I noticed yesterday that I am unable to transfer any out without the password. 

Looking into this, I noticed in the app UI that I have no user name set and in my Keystore json `meta.name` is `null`. I thought that was strange because I'm sure I would have set a name for my account if it had been requested during setup. Investigating the source code I can see that name is actually a required field in [createAccountForm (see line 96)](https://github.com/polkawallet-io/polkawallet-flutter/blob/f79fc6882b53011c027603ed4115a4632d112927/lib/page/account/create/createAccountForm.dart#L96). So I'm not sure how I even ended up with a null value for name.

In summary:

* I appear to have bypassed the name validator on createAccountForm suggesting I may have bypassed the whole form including setting a password too.
* I have no recollection of setting a name and password when setting up the app
* I am diligent with password management. I store all my passwords and mnemonics in LastPass.
* I'm a bit worried (to put it lightly) that I've lost a good portion of my financial assets here. 

I hope someone can please shed some light on this.

Thank you

**Comment by sheadawson** [comment]
Bump... Can anyone comment on this? I've just gone through the account creation step again and was not able to create an account without a username, so I have a feeling something went wrong when I created the original account. Also I notice biometrics is turned off on the account I'm locked out of. The account creation form ticks this option by default and I would have definitely kept it enabled if I was given the option as I use fingerprint access for everything that offers it.

Thanks for any help

**Comment by RomeroYang** [comment]
A username and password is always required to create an account. So the null value for your username may be some bug we never encountered and we will look into that.

For now, I hope your mnemonics were kept well so you can re-import your account with a new password.

**Comment by sheadawson** [comment]
@RomeroYang Thanks heaps for looking into it. Despite being very aware of how important the mnemonics are, I lost them. Combined with realising I don't appear to have a username/password/biometrics, very unfortunate for me :/

**Comment by achallans** [comment]
Had the same issue @sheadawson - did you manage to fix?

**Comment by sheadawson** [comment]
@achallans sorry to hear. No I didn't unfortunately, let me know if you do!

**Comment by achallans** [comment]
I have contacted them as it seems to have happened with the app requiring a password when we never set one...?!

**Comment by sheadawson** [comment]
@achallans when did you install the app and create the account? Wondering if there was a bug with our versions

**Comment by achallans** [comment]
Feb 19th 2021 - you? I honestly dont remember getting prompted to make an account

**Comment by sheadawson** [comment]
Earlier - 3 September 2020.

**Comment by achallans** [comment]
Interesting - I have contacted support and flagged the issue. Maybe try too and let me know

support@polkawallet.io
hello@polkawallet.io

I have also logged a bug on github

**Comment by JustforfunA** [comment]
@achallans @sheadawson hi, I had the same problem with my polkawallet. Support didn’t answer to me. Can you help me? how did you solve this problem

**Comment by achallans** [comment]
Didn’t resolve im afraid. Still waiting on a solution

**Comment by JustforfunA** [comment]
@achallans thanks for the answer, will wait and hope for the best
